### DISCUSSION TITLE
Installing Mitsuba-blender in Blender 3.6 LTS

### DISCUSSION THREAD

**Discussion by jmoraispk** [NVlabs/sionna#197] [General]
Hey there,

I had problems installing the mistuba plug-in in Blender 3.6. This is just to report the solution I found in case someone faces the same issue.

In the Sionna RT Sceen Creation [video](https://www.youtube.com/watch?v=7xHLDxUaQ7c), Fayçal mentioned mitsuba version 0.1. But when we go to [mitsuba-blender tags](https://github.com/mitsuba-renderer/mitsuba-blender/tags), we face the choice between "latest" and "v0.1". _Download the "latest" tag_ ([link](https://github.com/mitsuba-renderer/mitsuba-blender/releases/tag/latest)). Otherwise, mitsuba won't appear in add-ons after installing. 

The second crux is that the mitsuba export option may still not appear under the File -> Export menu. This may happen if you forgot to enable the add-on or if the add-on has missing dependencies. You can fix the problem with dependencies by opening the add-on in the add-on tab in Blender preferences, and click on "install dependencies", as mentioned in [mitsuba guides](https://github.com/mitsuba-renderer/mitsuba-blender/wiki/Installation-&-Update-Guide), and then restarting Blender.

General feedback: installation is a big barrier to entrance. It would be useful to include links in the docs to the exact tags and things to install; perhaps even making an installation video (for each supported operative system). 

Hopefully this saves time to someone. 
Thank you.

**Comment by faycalaa** [comment]
Hi,

Thank you for your feedback.

We have added the relevant links to the description of the [Sionna RT Scene Creation video](https://youtu.be/7xHLDxUaQ7c). Please note that most of these links can also be found in the [Introduction to Sionna RT tutorial](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Introduction.html).

Since there are already guides on the installation of Blender, Mitsuba-Blender, and Blender-OSM, we currently don't plan to publish additional content on this topic.

**Comment by merlinND** [comment]
Hi @jmoraispk,

As a side note, please be aware that the way geometry gets exported changed in Blender 3.4+. The mitsuba-blender addon needs to be updated to support this, which is tracked in this pull request: https://github.com/mitsuba-renderer/mitsuba-blender/pull/63

Until this is merged, we therefore recommend to stick with Blender 3.3 when exporting scenes to Mitsuba.

**Reply by jmoraispk** [reply]
Life saving advice. Thank you.
